x-pack/plugins/ingest_manager/README.md

```diff
@@ -15,13 +15,17 @@ See the Kibana docs for [how to set up your dev environment](https://github.com/
 
 
 One common development workflow is:
+ - Bootstrap Kibana
+     ```
+    yarn kbn bootstrap
+    ```
  - Start Elasticsearch in one shell
     ```
     yarn es snapshot -E xpack.security.authc.api_key.enabled=true
     ```
  - Start Kibana in another shell
     ```
-    yarn start --xpack.ingestManager.enabled=true --xpack.ingestManager.epm.enabled=true --xpack.ingestManager.fleet.enabled=true
+    yarn start --xpack.ingestManager.enabled=true --xpack.ingestManager.epm.enabled=true --xpack.ingestManager.fleet.enabled=true --no-base-path --xpack.endpoint.enabled=true
```

Comment:
What's the reason for adding this?

For context, I removed the `no-base-path` since it's not required and has lead to issues with things appearing to work locally and then failing for others or in CI.

I don't immediately see the value in the endpoint setting either, but it's not caused issues before.

Reply:
I think the problem here is that I started to mix 2 target groups. I recently get more and more asked on how to run Ingest Manager / Endpoint and this doc is the place I point them to. At the same time, this doc was initially created for our internal needed.

To solve the conflict, we should probably split it up into 2 docs, one for externals want to run Fleet and one for us.

Reply:
BTW: Agree on the --no-base-path issue. We should run it / test it in both modes but I don't want external tester / contributors to hit these issues.

Reply:
Question regarding endpoint being enabled - I think (but am not sure) that the Endpoint package is "automatically" installed, just like the System package, when the user visits the Ingest UI for the first time. If that is true, is that behaviour impacted by the wether the endpoint plugin is enabled or disabled?

Reply:
I think the current behaviour is that the Endpoint package is installed as soon as the Endpoint UI is opened for the first time (@jonathan-buttner probably knows more). So I think at the moment, it does not make a difference.

Reply:
@paul-tavares the endpoint package isn't installed by default yet, but it will be once my PR is merged. It won't affect anything if the endpoint plugin is enabled or disabled that I'm aware of. The user will just have some index patterns, dashboards, etc installed even if the plugin is disabled

Reply:
Thank you both
👍